packages/styles/scss/components/simplebenefits/_simplebenefits-item.scss

```diff
@@ -25,6 +28,8 @@
     }
 
     &__title {
+      overflow: hidden;
+      height: $spacing-11;
```

Comment:
@andysherman2121 We shouldn't be hard-coding this. Suggest switching to CSS `grid` and having it set the heights equally or we continue to use `sameHeight`.